WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -14,21 +14,23 @@ struct ItemListView: View {
 
     @Binding var selectedItemType: ItemType
 
+    var itemListController: PointOfSaleItemsControllerProtocol {
```

Comment:
Let's call selected ones `selected` or `current`, or it's easy to conflict the, with actual `itemsController`

```suggestion
    var selectedItemListController: PointOfSaleItemsControllerProtocol {
```

Reply:
It's probably clear enough now that the posModel only has specifically named controllers, no general `itemsController` – WDYT?

Reply:
Yes 👍 then it's clear.